core/string_utils.cpp

```diff
@@ -44,6 +44,7 @@ UString jsonnet_string_escape(const UString &str, bool single)
             case U'\n': ss << U"\\n"; break;
             case U'\r': ss << U"\\r"; break;
             case U'\t': ss << U"\\t"; break;
+            case U'\v': ss << U"\\v"; break;
```

Comment:
I think there is an issue here because this code is used in the JSON manifester and that will cause it to output invalid JSON because JSON does not have \v.
I think you can simply delete this line and everything will be fine, however, since it will fall back on the \u representation which will be fine.  Come to think of it, the explicit \0 case below this line is probably also unnecessary because the fallback will handle it!

I'm ok with adding \v to the Jsonnet language as long as we don't emit it as JSON.

Can you add \v into the tests in test_suite/unparse.jsonnet?  The golden would have \u000B instead of \v.

Feel free to ping me internally to discuss this (dcunnin@)

Reply:
Having said all that, it seems you haven't added \v to the go-jsonnet parser so that means the test you add in this PR will not work in go-jsonnet...

https://github.com/google/go-jsonnet/blob/master/internal/parser/string_util.go#L31

Maybe in that case what you actually want to do is abandon the adding of \v to Jsonnet and instead just use \U000b in your own code :)  It's up to you.

Reply:
I have updated to remove parsing of `\v` (as you mentioned it will cause output to be invalid JSON)